tests/unit/amo/reducers/test_reviews.js

```diff
@@ -1219,4 +1221,20 @@ describe(__filename, () => {
       expect(reviewsAreLoading({ reviews: state }, slug)).toBe(false);
     });
   });
+
+  describe('setViewContext', () => {
+    it('resets the view state', () => {
+      let state;
+
+      state = reviewsReducer(
+        undefined,
+        showEditReviewForm({
+          reviewId: 1,
+        }),
+      );
+      state = reviewsReducer(undefined, setViewContext(VIEW_CONTEXT_HOME));
```

Comment:
I thought about adding a check assertion here, to verify that `state.view` is not simply `{}` here, but we are already testing that in the test for `showEditReviewForm` above, so it's redundant.